src/deepsparse/benchmark/benchmark_sweep.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
```

Comment:
```suggestion
```

Reply:
Why though?

Reply:
Maybe cause it wouldn't interact with venvs properly?

Reply:
The env at the end works well with venvs, more details (here)[https://discuss.python.org/t/pep-394-clarify-bullet-for-usr-bin-env-python/7953]

Reply:
oh sweet, TIL!

Reply:
mostly thinking about being system dependent, but looks like that is at least somewhat covered based on the link you shared. fine for this since DS requires linux, but let's not add this in as a standard